src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-size: contain;
+  margin-right: 4px;
+}
```

Comment:
Consider specifying a fallback font-family for `.stars__star` in case the main font doesn't load. This ensures that the text remains legible and the layout consistent across different environments.